editor-components/Image_gallery/image_gallery_7/image-gallery7.tsx

```diff
@@ -0,0 +1,490 @@
+import * as React from "react";
+import { BaseImageGallery } from "../../EditorComponent";
+import styles from "./image-gallery7.module.scss";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+
+
+type ImageType = {
+    type: string;
+    key: string;
+    displayer: string;
+    value: string;
+  };
+  
+  
+  
+  type CardType = {
+    type: "array";
+    key: "card";
+    displayer: string;
+    value: ImageType[];
+  };
+  
+
+
+class ImageGallery7 extends BaseImageGallery {
+
+   
+    constructor(props?: any) {
+        super(props, styles);
+        
+        this.addProp({
+            type: "array",
+            key: "gallery",
+            displayer: "Gallery",
+            value: 
+                [
+                    {
+
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 1",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 1",
+                                value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669dfff22f8a5b002ce60115?alt=media", 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "Dubai",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Summer, Fashion"
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    },
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 2",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 2",
```

Comment:
use singular and non-contain number names.
```suggestion
                        displayer: "Image",
```